tox.ini

```diff
@@ -13,8 +14,13 @@ deps=
     django22: Django==2.2
     django31: Django==3.1
     django32: Django==3.2
+    djangomain: https://github.com/django/django/archive/main.tar.gz
 
 [testenv:py36-flake8]
 commands = flake8 templated_email tests --ignore=E501
 deps =
     flake8
+
+
+[testenv:{py38,py39}-djangomain]
+ignore_outcome = true
```

Comment:
Will we receive any notifications in case it fails?

Reply:
Good question. I wasn't able to understand if it does trigger notifications in the DRF repo because they use a codecov. I'm still not sure if this is something we'd have to configure in coveralls or elsewhere.

Reply:
Removed this, so failed tests will now get broken build emails form these cases